_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title:  "How to build your own KafkaConnect image via custom resource"
+date: 2021-03-23
+author: jakub_stejskal
+---
+In one of [the older blog post](https://strimzi.io/blog/2020/05/07/camel-kafka-connectors/) we shown you how easy is to integrate [Camel Kafka Connectors](https://camel.apache.org/camel-kafka-connector/latest/) with Strimzi via `KafkaConnect` custom resource.
+That approach had one limitation - you had to build your own Kafka Connect image and use it in custom resource.
+However, this step is not needed anymore, thanks to feature which we introduced in Strimzi 0.22 to allow users to build custom Kafka Connect images directly by Strimzi.
+
+### How does it work
+
+Strimzi uses two different ways how to build the image based on the underlying Kubernetes cluster:
+* In case you use Kubernetes, the image will be built by [kaniko](https://github.com/GoogleContainerTools/kaniko)
+* In case you use Openshift, the image will be built by Openshift Builds
+
+Let's describe kaniko by quoting description from it's github:
+> kaniko is a tool to build container images from a Dockerfile, inside a container or Kubernetes cluster.
+> kaniko doesn't depend on a Docker daemon and executes each command within a Dockerfile completely in userspace.
+> This enables building container images in environments that can't easily or securely run a Docker daemon, such as a standard Kubernetes cluster.
```

Comment:
I would probably put it like this. But that is just my personal preference:

```suggestion
Strimzi uses two different ways how to build the image based on the underlying Kubernetes cluster.
In case you use Kubernetes, the image will be built by [kaniko](https://github.com/GoogleContainerTools/kaniko).
kaniko is a tool to build container images from a Dockerfile, inside a container or Kubernetes cluster.
kaniko doesn't depend on a Docker daemon and executes each command within a Dockerfile completely in userspace.
This enables building container images in environments that can't easily or securely run a Docker daemon, such as a standard Kubernetes cluster.
In case you use Openshift, the image will be built by Openshift Builds
```